language/stdlib/modules/libra_coin.mvir

```diff
@@ -35,11 +35,11 @@ module LibraCoin {
 
         release(move(capability));
 
-        // TODO: temporary measure for testnet only: limit minting to 1B coins at a time.
+        // TODO: temporary measure for testnet only: limit minting to 1B Libra at a time.
         // this is to prevent the market cap's total value from hitting u64_max due to excessive
         // minting. This will not be a problem in the production Libra system because coins will
         // be backed with real-world assets, and thus minting will be correspondingly rarer.
-        assert(copy(value) <= 1000000000, 11);
+        assert(copy(value) <= 1000000000 * 1000, 11); // * 1000 because the unit is microlibra
```

Comment:
```suggestion
        assert(copy(value) <= 1000000 * 1000000, 11); // * 1000000 because the unit is microlibra
```

Reply:
Too bad we do not have quote as digit separator.
In this case, 1B libra should be:
1'000'000'000 * 1'000'000